.github/workflows/main.yml

```diff
@@ -15,10 +18,21 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-west-1
+      - name: Transform ECR Image name
+        if: startsWith(github.base_ref, 'stable') || startsWith(github.base_ref, 'master')
```

Comment:
What does this conditional do?

Reply:
Right, Matt. We are conditionally building on these branches but just to execute this task on pull request event only, I have added this additional check. If it would be any other event like push or create then we won't get the **base_ref** in the github context and it will lead to failing of the workflow. So to avoid any failure scenarios I have added this conditional.